### PR TITLE
The rest of the slash command conversions

### LINKED ISSUE #54: Convert rest of commands to slash commands
Progress has been made to convert old commands to slash commands, but there's still more to convert.

Conversion status:
 - [x] affirmation (_easy_)
 - [x] catfacts (_easy_)
 - [x] compliment (_easy_)
 - [x] counterstrike (_easy_)
 - [x] course (_moderate_)
 - [x] dadjoke (_easy_)
 - [x] exams (_moderate_)
 - [x] insult (_easy_)
 - [x] listsound (_moderate_)
 - [x] meow
 - [x] quote (_moderate_)
 - [x] random (_easy_)
 - [x] rigged (_easy_)
 - [x] room (_moderate_)
 - [x] slap
 - [x] sound (_moderate_)
 - [x] spook (_moderate_)
 - [x] ta (_moderate_)
 - [x] updateexams (_moderate_)
 - [x] wednesday (_easy_)
 - [x] winner (_moderate_)

### PR DESCRIPTION
**Pull request by andesyv**
This one builds on previous work done in #57, so do that one first.

Because Discord interactions allows for it, whenever the user gets a reply that doesn't need to be shown to everyone (like wrong usage of a command or an error), the bot will only show the respond to the user.

~~Currently all commands are available to everyone. You can mark commands as not accessible to everyone, but we will have to then find a way to manually add them to admins ourselves. (Like with a `hidden` flag that manually PUT's them to the roles in `registerCommand`)~~

This closes #54.

### CODE REVIEW

File: src/client/config.ts [hunk 1]
```diff
@@ -20,5 +23,8 @@ export const getConfig = (): Config => {
       appId: getEnvVar("DISCORD_APP_ID"),
       guildId: getEnvVar("DISCORD_GUILD_ID"),
     },
+    guild: {
+      adminRoles: ["Admin", "Orakel"],
```
Comment (sondr3) [inline src/client/config.ts:27]: What?

File: src/commands/sound/listsound.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      temp += arg + "\n";
+    } else {
+      snippets.push(temp);
+      temp = "";
+    }
+  }
+  if (temp.length !== 0) snippets.push(temp);
+
+  return snippets;
+};
```
Comment (sondr3) [inline src/commands/sound/listsound.ts:38]: image

Comment (andesyv) [reply src/commands/sound/listsound.ts:38]: Is there already a built-in function that does this? Concatinating a list into segments of a max size.

Comment (sondr3) [reply src/commands/sound/listsound.ts:38]: Aner ikke, bare legg gjerne inn en kommentar på hva denne funksjonen gjør. Hjernen min smelter av å se på den.

**Review by sondr3 — APPROVED** [review]

**Pull request merged**